internal/resultstore/resultstore.go

```diff
@@ -21,8 +22,7 @@ import (
 )
 
 type ResultStore struct {
-	bucket        string
-	basePath      string
```

Comment:
I see that `basePath` has been removed. Was there a reason to eliminate it?

It may also create issues with cloud storage. For example: "gs://bucket/prefix/to/files" was previously supported, but will no longer behave as expected for GCS and S3 buckets, as "prefix/to/files" is ignored by `blob.OpenBucket`.

Unfortunately for `file` schemes the path is not ignored, so it gets complicated.

Reply:
I removed it because I thought that the URL object could handle it with its `Path` attribute, but as you say, the behaviour for `file` schemes is a special case. I'll reinstate it but might rename it to something like `prefix`.